### ISSUE TITLE
Show page under Table of Contents

### ISSUE THREAD

**Issue by pmangahis** [pkp/pkp-lib#7557]
In OJS 2, the TOC allowed JM to assign page numbers. With more journals opting to publish in a continuous publication model (with the current workaround), it would be helpful to have this feature again.

When a JM adds an article to a published issue it's not always easy to decern where it ends up in the lineup. This is especially harder with larger issue line ups.

Requested by JM using OJS 3.3. 

![image](https://user-images.githubusercontent.com/50342731/146608307-12094e2f-1782-439e-a94a-e33d637fdc54.png)

**Comment by NateWr** [comment]
The page data can be entered at Publication > Issue > Page and this is shown in the table of contents on the default theme. Are you maybe using a different theme that doesn't show this information?

**Comment by pmangahis** [comment]
Yup! Aware of this. The JM is hoping to have the page numbers show in the back end TOC  to allow them to easily rearrange articles that get added to the issue.

![image](https://user-images.githubusercontent.com/50342731/146822963-5413140b-b3b3-436d-995c-7026ca1c50cf.png)

**Comment by NateWr** [comment]
Ah! I understand now, thanks!

**Comment by pmangahis** [comment]
+ 2 from PS hosted journals
